### PR TITLE
Update [ScoreCard] component (docs & examples)

### PR DESCRIPTION
**Pull request by manojleaton**
Fixes #BLUI-4374 .

#### Changes proposed in this Pull Request:
- Updated ScoreCard component with MD3Theme.
- Please verify this PR for kitchen sink example updates for ScoreCard component [ScoreCard KitchenSink](https://github.com/etn-ccis/blui-react-native-showcase-demo/pull/90)

#### Screenshots / Screen Recording (if applicable)
Screenshot 2023-10-16 at 2 36 12 PM

#### To Test:
- yarn start

#### Any specific feedback you are looking for?
-

### CODE REVIEW

File: components/src/md3/ScoreCard/ScoreCard.tsx [hunk 1]
```diff
@@ -0,0 +1,412 @@
...
+    StyleSheet,
+    ImageSourcePropType,
+    Image,
+    TouchableOpacity,
+    StyleProp,
+    ViewStyle,
+    ImageStyle,
+    TextStyle,
+} from 'react-native';
+import * as Typography from '../../core/typography';
```
Comment (ArshdeepSingh3-eaton) [inline components/src/md3/ScoreCard/ScoreCard.tsx:13]: Replace Typography with Text Component

Comment (ArshdeepSingh3-eaton) [reply components/src/md3/ScoreCard/ScoreCard.tsx:13]: Please Make sure to Not import anything from core folder

**Review by ektaghag-eaton — COMMENTED** [review]
1. Fontsize for title, subtitle, and info doesn't match with [Figma](https://www.figma.com/file/Nydz8n8HBj6U3ZtULja3gD/%F0%9F%92%AB-Brightlayer-UI-Tokens-%2B-Components-(in-Material-Design-3)?type=design&node-id=777-18275&mode=design&t=gNVKKVtLy2vtjZIh-0)
2. ScoreCard Header size is 112px currently it is 100px as per Figma
3. Icon size breaks when passing iconSize={48}
Screenshot 2023-10-13 at 1 30 14 PM

We need to test the Score Card again once InfolistItem and HeroBanner are available in the MD3 Theme

**Comment by huayunh** [comment]
the corner radius of the four corners are different

And please give it a default background color that is not pure white. I recommend "surface container low"

**Comment by huayunh** [comment]
Please update the header background color to use that "primary container" color by default

image

**Comment by manojleaton** [comment]
Added space between Buttons and Stepper as mentioned in the demo. I have updated that change in this PR itself since it's a minor change. Please verify.
Screenshot 2023-10-16 at 5 23 09 PM

**Comment by huayunh** [comment]
I played with it on the design side and actually found 24px to be more comfortable
Do you mind making it 24 instead?

Here's what it would look like on the design side:

image

**Comment by manojleaton** [comment]
> I played with it on the design side and actually found 24px to be more comfortable Do you mind making it 24 instead?
> 
> Here's what it would look like on the design side:
> 
> image

have added 24px padding, please check

Screenshot 2023-10-16 at 8 44 20 PM

**Comment by huayunh** [comment]
And can you adjust the scorecard header padding too? I think this was redesigned a bit to fit with the new typography. 

The vertical padding is now 24; horizontal padding shall stay at 16

**Comment by joebochill** [comment]
It looks like something strange is happening with the padding/margin between the stepper and the buttons...your screenshot when you had it set to 16 it looked like 8, when you had it set to 24 it looks like 16...is there some overlap happening?

**Comment by joebochill** [comment]
I think the issue with the MobileStepper is more an issue with overflow. The flexing of the containers and the spacing looks to be working as expected (at least on my machine), but the dots are just overflowing out of the container when there are too many of them, making it look like the spacing is broken.

I think the more robust solution would be to make `stepperContainer` flex:0 when the mode is text or dots and flex: 1 when the mode is progress. There will still be issues with overflow if there are too many dots (not new), but the flex behavior will I think be a bit more appropriate (we can still remove the flex wrapper around the buttons in the example, but it should also work if the flex wrapper IS there). What do you think?

**Comment by manojleaton** [comment]
> I think the issue with the MobileStepper is more an issue with overflow. The flexing of the containers and the spacing looks to be working as expected (at least on my machine), but the dots are just overflowing out of the container when there are too many of them, making it look like the spacing is broken.
> 
> I think the more robust solution would be to make `stepperContainer` flex:0 when the mode is text or dots and flex: 1 when the mode is progress. There will still be issues with overflow if there are too many dots (not new), but the flex behavior will I think be a bit more appropriate (we can still remove the flex wrapper around the buttons in the example, but it should also work if the flex wrapper IS there). What do you think?

Please refer this PR for the changes as per comment [MobileStepper](https://github.com/etn-ccis/blui-react-native-component-library/pull/415)

File: components/src/md3/Hero/Hero.tsx [hunk 2]
```diff
@@ -29,8 +29,8 @@ const makeStyles = (theme: MD3Theme, fontScale: number): StyleSheet.NamedStyles<
             alignItems: 'center',
             justifyContent: 'center',
             overflow: 'hidden',
-            height: 36 * fontScale,
-            width: 36 * fontScale,
+            // height: typeof iconSize === 'number' ? normalizeIconSize() : iconSize * fontScale,
```
Comment (ektaghag-eaton) [inline components/src/md3/Hero/Hero.tsx:32]: Please remove commented code

**Review by ektaghag-eaton — APPROVED** [review]

**Review by JeffGreiner-eaton — APPROVED** [review]
looks good, just Ekta's comment

**Pull request merged**